.github/workflows/main.yml

```diff
@@ -12,6 +13,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v1
+      - uses: Mushus/checkout-submodule@v1
```

Comment:
I think in the checkout one you can ask for the submodules directly. How do we do it in jormungandr?

Reply:
will try with actions/checkout@v2